scripts/install

```diff
@@ -6,8 +6,27 @@ source _ynh_mysql_connect_as.sh
 
 phpflags="--define apc.enable_cli=1"
 
+#=================================================
+# STORE SETTINGS FROM MANIFEST
+#=================================================
+
+ynh_app_setting_set --app=$app --key=maintenance --value=$maintenance_mode
```

Comment:
```suggestion
ynh_app_setting_set --app=$app --key=maintenance --value=$maintenance
```

Reply:
we get this warning with the CI `WARNING ./install: line 13: maintenance: unbound variable`